components/introduction/modules/ROOT/pages/index.adoc

```diff
@@ -11,28 +85,27 @@ video::Anb0ZSruWlw[youtube, height=450, width=800, opts="modest"]
 
 See more videos here: https://www.youtube.com/skale
 
-== SKALE Chain Sizes
-
-SKALE Chains can be configured into different sizes - sizes being the amount of resource allocation that each of the 16 nodes provides. xref:skale-chain-sizes.adoc[Read more here].
 
-== SKALE Chain FUEL (sFUEL)
+== Tools & Compatibility
 
-SKALE Chains operate with sFUEL, which unlike ETH has no monetary value, but like ETH is used to conduct transactions. xref:skale-chain-fuel.adoc[Learn more about managing sFUEL on your SKALE Chain].
+SKALE Network is designed to work with all Ethereum-compatible tools such as API-based wallets and monitoring and analytics. 
```

Comment:
Suggested change

...non-custodial wallets, chain agnostic API-based wallets, or those with dedicated support  for the SKALE Network, monitoring platforms, and analytic solutions.